media/css/mozorg/contribute/contribute-2015.less

```diff
@@ -958,6 +962,53 @@ textarea:focus {
     }
 }
 
+.inquiry-choices {
+    .option-group {
+        text-align: center;
+
+        h2 {
+            .font-size(24px);
+        }
+    }
+
+    ul.option-list {
+        list-style: none;
+        text-align: center;
+
+        li.option {
+            display: inline-block;
+            vertical-align: top;
+            float: none;
+            width: 210px;
+        }
+    }
+
+    .option-double {
+        width: 440px;
+        display: inline-block;
+        text-align: left;
```

Comment:
If you would like to fix the alignment when there's two items, change this to `text-align: center;`